code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -274,8 +274,8 @@
 
 
 
+GLOBAL_VAR(blackbox)
```

Comment:
Document. Also, make it typed, whatever the macro for that was.

Reply:
yeeet